index.js

```diff
@@ -28,7 +28,7 @@ function assertListener(listener) {
 function getListeners(instance, eventName) {
 	const events = eventsMap.get(instance);
 	if (!events.has(eventName)) {
-		events.set(eventName, new Set());
+		return null;
```

Comment:
```suggestion
		return;
```

I prefer not using `null`.